### ISSUE TITLE
Test generating jupyter notebooks for Python with Jupinx

### ISSUE THREAD

**Issue by jlperla** [ubcecon/computing_and_datascience#61]
See https://medium.com/quantecon-blog/introducing-jupinx-60ba9fc12f4f
and
https://github.com/QuantEcon/sphinxcontrib-jupyter
and
https://github.com/QuantEcon/sphinxcontrib-jupyter.minimal

**Comment by jlperla** [comment]
Sorry @jasmineyang   This is the last one to try to figure out on the python notebooks.

**Comment by jlperla** [comment]
You can add in files into https://github.com/ubcecon/computing_and_datascience/tree/master/python_sandbox to test things out

**Comment by jlperla** [comment]
@ArnavSood  I was hoping that you could take a look at this.  I tried to do it on Windows and had trouble (you can see my report in https://github.com/QuantEcon/sphinxcontrib-jupyter.minimal/issues/2 )

Give it a quick shot on windows, and if that fails then install an Ubuntu dual-boot or a VM or something like that to try it on linux.

**Comment by ArnavSood** [comment]
@jlperla OK, I got this to work. My Windows is a mess so I just did this on my OS X. Only thing I needed to change was `python` -> `python3` in the Makefile, since my python points at 2.7. 

Next steps are (a) documenting this/putting the files in the sandbox, and (b) updating my Windows install and getting it to work there, too.

**Comment by jlperla** [comment]
I want to get to the point where you can reproduce my problem on windows (if it doesn't work). So take your time on getting your windows up and running

**Comment by ArnavSood** [comment]
Agree. Will keep you posted.

**Comment by ArnavSood** [comment]
FYI, was unable to reproduce error on Ubuntu on the work machine. I had some difficulty working with Windows, but I will try and duplicate your setup. 

Here's what I did: 

1. Download official conda installer (`Anaconda3-5.2.0-Linux_x86_64.sh`). 
2. Build per tutorial.
3. Run a `conda update -all`. Lots of things were updated. 
4. Clone repos and setup. 
5. Download `make` and run `make jupyter` on the minimal repo.
